src/tests/integration/test-data/foundry-prove-skip-legacy

```diff
@@ -62,6 +62,8 @@ BytesTypeTest.testFail_bytes32(bytes32)
 BytesTypeTest.testFail_bytes4(bytes4)
 ChainIdTest.test_chainid_setup()
 CoinBaseTest.test_coinbase_setup()
+ConstructorTest.test_constructor()
+ConstructorTest.run_constructor()
```

Comment:
This `foundry-prove-skip-legacy` file is [used](https://github.com/runtimeverification/kontrol/blob/134b0a407385b769727bc55f291c3164782ee119/src/tests/integration/test_foundry_prove.py#L147) to determine which tests to not run with legacy backend in `test_foundry_prove`, and is not used in `test_foundry_init_code`. 

At the same time, all [`test_foundry_init_code`](https://github.com/runtimeverification/kontrol/blob/c1104375d608b20496a2c94435a34f82a16e28c9/src/tests/integration/test_foundry_prove.py#L576
) tests are being skipped in the legacy backend job:
```
def test_foundry_init_code(test: str, foundry: Foundry, bug_report: BugReport | None, no_use_booster: bool) -> None:
    if no_use_booster:
        pytest.skip()
```
If that's intentional, you can just remove these `ConstructorTest` tests from `foundry-prove-skip-legacy`. If you'd prefer to run some of the tests on the legacy backend (which might be reasonable if they aren't taking more than 300s to execute there — that's the threshold we currently use), you can do something similar to
```
    if (no_use_booster and test_id in SKIPPED_LEGACY_INIT_CODE_TESTS)):
        pytest.skip()
```
where `SKIPPED_LEGACY_INIT_CODE_TESTS` can be read from a file such as `foundry-init-code-skip`.